src/components/withKeyboardState.tsx

```diff
@@ -73,12 +73,12 @@ function KeyboardStateProvider({children}: ChildrenProps): ReactElement | null {
             onStart: () => {
                 'worklet';
 
-                runOnJS(setIsKeyboardAnimating)(true);
+                scheduleOnRN(setIsKeyboardAnimating, true);
             },
             onEnd: () => {
                 'worklet';
 
-                runOnJS(setIsKeyboardAnimating)(false);
+                scheduleOnRN(setIsKeyboardAnimating, true);
```

Comment:
Previously this line was `runOnJS(setIsKeyboardAnimating)(false);` so it should be `false` instead of `true`.
```suggestion
                scheduleOnRN(setIsKeyboardAnimating, false);
```